js/swipe.js

```diff
@@ -349,6 +349,7 @@ function Swipe(container, options) {
 
   function begin() {
 
+    if (interval != null) clearTimeout(interval);
```

Comment:
I think this should be `interval !== null` (strict comparison), shouldn't it?

Reply:
You are right. "interval" is "undefined" at the beginning, so you could check for (interval !== undefined).

To be as strict as possible I am setting interval to null at the beginning and set it to null after the timeout triggered.